libs/execution_context/ebpf_protocol.h

```diff
@@ -9,11 +9,17 @@
 
 typedef enum _ebpf_operation_id
 {
+#if !defined(CONFIG_BPF_JIT_DISABLED)
```

Comment:
nit: removing the operation ids from the middle will cause the value of the other ids to change. So the same operation ID will have different value in native-only vs. regular build. Not sure if this can cause any issues.

Reply:
Since the IDs are not used in apps (they should be calling libbpf) it's ok to have breaking changes in my view